scripts/empty_upgrade_handler_gen.sh

```diff
@@ -1,64 +1,65 @@
 #!/bin/bash
 
 # 1) this script creates an empty directory in app/upgrades called "vX" where X is a previous version + 1 with an empty upgrade handler.
-# 2) increases E2E_UPGRADE_VERSION in makefile by 1
-# 3) adds new version to app.go
+# 2) adds new version to app.go
+# 3) increases E2E_UPGRADE_VERSION in makefile by 1
+# 4) bumps up previous e2e-init version in tests/e2e/containers/config.go
 
 # Also insures that all the imports make use of a current module version from go mod:
 # (see:    module=$(go mod edit -json | jq ".Module.Path")      in this script)
 # Github workflow which calls this script can be found here: osmosis/.github/workflows/auto-update-upgrade.yml
 
- latest_version=0
```

Comment:
What are these changes?

Reply:
Hmm, seems like I merge this branch with main branch on notional. Let me fix them.